RecoHGCal/TICL/plugins/PatternRecognitionbyCAConstants.h

```diff
@@ -0,0 +1,15 @@
+#ifndef __RecoHGCal_TICL_PRbyCAConstants_H__
+#define __RecoHGCal_TICL_PRbyCAConstants_H__
+
+namespace ticl {
+  namespace patternbyCA {
```

Comment:
just for syntax, this can just be ```namespace ticl::patternbyCA {```

namespaces should be in all lower case.
Will these constants and typedef ever be used without PatternRecognitionByCA.h already included?
If no, perhaps it's more expressive to put them inside the class PatternRecognitionByCA.
OTOH, it seems like the use case covers more than just PatternRecognitionByCA and perhaps a less dual naming convention is better.

Reply:
Indeed they are used in the HGCGraph.